### PR TITLE
remove django dependency

### PR DESCRIPTION
**Pull request by jairhenrique**

### CODE REVIEW

**Comment by rpedigoni** [comment]
😍

File: tests/test_pool.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ class FakeXYZBackend(ThreadSafeCreateMixin):
 
 class TestBackendPool(object):
 
-    def test_get_backends_should_return_all_backends_instances(self, settings):
```
Comment (RomuloOliveira) [inline tests/test_pool.py]: What do you think about adding a `settings` fixture to `conftest.py`?

```python
import copy

import pytest

from ramos import compat

@pytest.fixture
def settings():
    original_settings = copy.deepcopy(compat.settings)
    yield compat.settings
    compat.settings = original_settings
```

Comment (jairhenrique) [inline tests/test_pool.py]: I tried this approach, but I got some issues. Can I try this in another moment? 🤔

File: tests/test_compat.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+from ramos.compat import import_string
+
+
+class TestImportString(object):
+
+    def test_should_raise_exception_when_path_has_no_dot(self):
+        with pytest.raises(ImportError):
+            import_string('no_dots_in_path')
+
+    def test_should_raise_exceptions_when_module_has_no_attribure(self):
```
Comment (rschalch-luizalabs) [inline tests/test_compat.py]: attribute

Comment (jairhenrique) [inline tests/test_compat.py]: 👍

**Pull request merged**